apps/content-publishing-api/src/api.config.spec.ts

```diff
@@ -12,6 +12,7 @@ describe('Content Publishing API Config', () => {
     API_PORT: undefined,
     // API_TIMEOUT_MS: undefined,
     FILE_UPLOAD_MAX_SIZE_IN_BYTES: undefined,
+    FILE_UPLOAD_COUNT_LIMIT: undefined,
```

Comment:
Should also add 2 test cases for `validateMissing` and `shouldFailBadValues` in the 'invalid environment' section, since this is a required config.